packages/web/src/index.css

```diff
@@ -159,8 +159,7 @@ body {
 
 html,
 body,
-#root,
-#root > div {
```

Comment:
Note: This change means `App`, `ToastContext`, `SomethingWrong`, `RequiresUpdate` and any other divs directly descending from root (please tell me if there's more, I probably missed one) need to _set these styles themselves_, but I ultimately think this is best left to be "non-magical"